addons/web/static/src/scss/bootstrap_overridden_frontend.scss

```diff
@@ -37,6 +37,7 @@ $body-emphasis-color: $black !default; // BS Default
 $border-width: 1px !default; // BS Default
 
 $component-active-bg: $primary !default;
+$component-active-color: color-contrast($component-active-bg) !default;
```

Comment:
I think there is a bit more to do. When you click a bunch of letter "disappear" as they have the same color as the active element background:

![image](https://github.com/user-attachments/assets/2dce8fe6-b2dc-450c-bb45-5dc1d413ff3e)